File: app/src/main/java/com/yoesuv/infomadiun/fragment/ChildFragmentThanks.java
package com.yoesuv.infomadiun.fragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.text.Html;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.yoesuv.infomadiun.R;


public class ChildFragmentThanks extends Fragment {

    private View v;
    private TextView tvThanks;

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        v = inflater.inflate(R.layout.activity_thanks, container, false);

        tvThanks = (TextView) v.findViewById(R.id.textView_terima_kasih);
        tvThanks.setText(Html.fromHtml(getResources().getString(R.string.trims)));
        return v;
    }
}

File: app/src/main/java/com/yoesuv/infomadiun/menu/other/views/ChildFragmentThanks.kt
package com.yoesuv.infomadiun.menu.other.views

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.databinding.DataBindingUtil
import androidx.fragment.app.Fragment
import com.yoesuv.infomadiun.R
import com.yoesuv.infomadiun.databinding.ChildFragmentThanksBinding
import com.yoesuv.infomadiun.utils.AppHelper

class ChildFragmentThanks : Fragment() {
    companion object {
        fun getInstance(): Fragment = ChildFragmentThanks()
    }

    private lateinit var binding: ChildFragmentThanksBinding

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?,
    ): View {
        binding = DataBindingUtil.inflate(inflater, R.layout.child_fragment_thanks, container, false)
        binding.textViewThanks.text = AppHelper.fromHtml(getString(R.string.trims))
        return binding.root
    }
}
